src/shared/scrapers/USA/WV/index.js

```diff
@@ -77,43 +77,85 @@ const scraper = {
     'Wyoming County'
   ],
 
-  async scraper() {
-    const $ = await fetch.page(this.url);
+  scraper: {
+    '0': async function() {
+      const $ = await fetch.page(this.url);
 
-    const $p = $('p:contains("Counties with positive cases")');
+      const $p = $('p:contains("Counties with positive cases")');
 
-    const list = $p
-      .text()
-      .split(':')[1]
-      .split(',');
+      const list = $p
+        .text()
+        .split(':')[1]
+        .split(',');
 
-    let counties = [];
+      let counties = [];
 
-    for (const item of list) {
-      const items = item.split('(');
-      if (items.length !== 2) {
-        continue;
-      }
+      for (const item of list) {
+        const items = item.split('(');
+        if (items.length !== 2) {
+          continue;
+        }
+
+        const county = geography.addCounty(parse.string(items[0]));
+        let cases = items[1];
 
-      const county = geography.addCounty(parse.string(items[0]));
-      let cases = items[1];
+        if (cases.slice(-1) !== ')') {
+          continue;
+        }
 
-      if (cases.slice(-1) !== ')') {
-        continue;
+        cases = parse.number(cases.slice(0, -1));
+
+        counties.push({
+          county,
+          cases
+        });
       }
 
-      cases = parse.number(cases.slice(0, -1));
+      counties = geography.addEmptyRegions(counties, this._counties, 'county');
+      counties.push(transform.sumData(counties));
 
-      counties.push({
-        county,
-        cases
-      });
-    }
+      return counties;
+    },
+    '2020-04-03': async function() {
+      this.url =
+        'https://wabi-us-gov-virginia-api.analysis.usgovcloudapi.net/public/reports/187b4de8-78ef-40be-9510-7fa9a1ef89f2/modelsAndExploration';
+      const options = {
+        headers: { 'X-PowerBI-ResourceKey': '187b4de8-78ef-40be-9510-7fa9a1ef89f2' }
+      };
```

Comment:
I guess there's no real way to check that these don't change.

Reply:
If they change, the rest of this scraper will explode. They've been working for ~16 hours  so I'm hoping it's not a session that will expire.